presets.js

```diff
@@ -0,0 +1,217 @@
+const { combineRgb } = require('@companion-module/base');
+
+module.exports = function (self) {
+    const presets = {};
+
+    // Função para criar presets de PPT
+    const createPptPreset = (index) => {
+        return {
+            type: 'button',
+            category: 'PPT',
+            name: 'pr_init',
+            style: {
+                text: `PPT ${index}`,
+                size: 'auto',
+                color: combineRgb(255, 255, 255),
+                bgcolor: combineRgb(128, 0, 0),
+            },
+            steps: [
+                {
+                    down: [
+                        {
+                            actionId: 'send_int',
+                            options: {
+                                path: `/ppt${index}`,
+                                value: 1,
+                            },
+                        },
+                    ],
+                    up: [],
+                },
+            ],
+            feedbacks: [],
+        };
+    };
+
+    // Função para criar presets de VIDEO
+    const createVideoPreset = (index) => {
+        return {
+            type: 'button',
+            category: 'VIDEO',
+            name: 'pr_init',
+            style: {
+                text: `VIDEO\n${index}`,
+                size: 'auto',
+                color: combineRgb(255, 255, 255),
+                bgcolor: combineRgb(25, 25, 200),
+            },
+            steps: [
+                {
+                    down: [
+                        {
+                            actionId: 'send_int',
+                            options: {
+                                path: `/video${index}`,
+                                value: 1,
+                            },
+                        },
+                    ],
+                    up: [],
+                },
+            ],
+            feedbacks: [],
+        };
+    };
+
+    // Cria presets de PPT de 01 a 20
+    for (let i = 1; i <= 20; i++) {
+        let num = i < 10 ? `0${i}` : `${i}`;
+        presets[`PPT${num}`] = createPptPreset(num);
+    }
+
+
+
+    // Cria presets de VIDEO de 01 a 10
+    for (let i = 1; i <= 10; i++) {
+        let num = i < 10 ? `0${i}` : `${i}`;
+        presets[`VIDEO${num}`] = createVideoPreset(num);
+    }
+
+    // Presets adicionais
+    presets['ESC'] = {
+        type: 'button',
+        category: 'PPT',
+        name: 'pr_init',
+        style: {
+            text: 'ESC',
+            size: 'auto',
+            color: combineRgb(255, 255, 255),
+            bgcolor: combineRgb(0, 0, 120),
+        },
+        steps: [
+            {
+                down: [
+                    {
+                        actionId: 'send_int',
```

Comment:
You are using the generic send_int action for a process, which doesn't need an integer at all and which should have its own action.